Makefile

```diff
@@ -594,7 +593,8 @@ glance_cleanup: ## deletes the operator, but does not cleanup the service resour
 
 .PHONY: glance_deploy_prep
 glance_deploy_prep: export KIND=Glance
-glance_deploy_prep: export IMAGE=${GLANCEAPI_DEPL_IMG}
+glance_deploy_prep: export IMAGE=${GLANCEAPI_DEPL_IMG},${GLANCEAPI_DEPL_IMG},${GLANCEAPI_DEPL_IMG}
```

Comment:
So let's see if I understand this properly:
- do we need the image export here because the `ContainerImage` is not "inherited" from the top-level CR? Or the purpose here is to enforce different images for multiple glance instances? I see this pattern limiting if I grow (in the future) the number of glance-api instances (that can happen for many purposes)

Reply:
> do we need the image export here because the ContainerImage is not "inherited" from the top-level CR? 

That is correct. We have to explicitly set containerImage spec in all layers because the one in base Glance CR is not inherited to the sub Glance(Internal|External)CR.

>  I see this pattern limiting if I grow (in the future) the number of glance-api instances (that can happen for many purposes)

If you add a new instance then you can add that to IMAGE and IMAGE_PATH. I don't fully understand what "limits" adding an instance. Do you mind elaborating your concern ?

Reply:
no big concerns, sorry for the noise, I think I understand the purpose of your export here and actually there's no limit as this part is only limited to the `standalone` glance deploy.